Documentation/CRDs/ceph-client-crd.md

```diff
@@ -41,3 +41,121 @@ spec:
 ### Prerequisites
 
 This guide assumes you have created a Rook cluster as explained in the main [Quickstart guide](../Getting-Started/quickstart.md).
+
+## Example
+
+### Using a `CephClient` to create Ceph credentials
+
+To use `CephClient` to connect to a Ceph cluster:
+
+#### 1. Create the `CephClient` (`metadata.name` will be your Ceph client ID)
+
+Apply your CephClient resource YAML (see above for an example client config) with `kubectl`:
+
+```console
+kubectl apply -f example.cephclient.yaml
```

Comment:
Where is this example.cephclient.yaml file? It might be best to show the example using the `ceph-client.yaml` file that has cinder and glance users.

Reply:
Oh I might need some feedback on that -- I don't think I made one that has *both* kinds of users. Will add this, and would appreciate some feedback!

Reply:
Similar to `ceph-client.yaml` in the examples folder, you could create a file such as `sqlite-client.yaml`

Reply:
Done! I added a new file with the example from the issue comments along with some explanatory comments.